ttnn/cpp/ttnn/operations/conv/conv2d/device/conv2d_op.cpp

```diff
@@ -58,7 +58,7 @@ Tensor optimized_conv_new(const Tensor& a, const Tensor &b, std::optional<const
     DataType dtype,
```

Comment:
there is no point in passing in here both math_fidelity and compute_kernel_config

Reply:
Removed math fidelity.